assets/src/scripts/interactive/pages/review-request.jsx

```diff
@@ -34,28 +33,48 @@ function ReviewRequest() {
   }
 
   const dataForSubmission = () => {
-    const { codelist0, codelist1, codelistA, codelistB, ...data } = formData;
-
-    return {
-      ...data,
+    const {
+      codelistA,
+      timeOption,
+      timeScale,
+      timeValue,
+      filterPopulation: filterOpts,
+      demographics: demographicOpts,
+      title,
+      purpose,
+    } = formData;
+
+    return removeUndefinedValuesFromObject({
       codelistA: {
         label: codelistA.label,
         type: codelistA.type,
         value: codelistA.value,
       },
       codelistB: {
-        label: codelistB.label,
-        type: codelistB.type,
-        value: codelistB.value,
+        label: codelistA.label,
```

Comment:
Should these three be codelistB rather than codelistA?

Reply:
Great spot! Copy and paste fail there, I'll fix now.